packages/builder/src/pages/builder/_layout.svelte

```diff
@@ -16,6 +17,32 @@
   $: user = $auth.user
 
   $: useAccountPortal = cloud && !$admin.disableAccountPortal
+  let showVerificationPrompt = false
+
+  const checkVerification = user => {
+    if (!showVerificationPrompt && user?.account?.verified === false) {
+      showVerificationPrompt = true
+      banner.queue([
+        {
+          message: `Please verify your account. We've sent the verification link to ${user.email}`,
+          type: BANNER_TYPES.NEUTRAL,
+          showCloseButton: false,
+          extraButtonAction: () => {
+            fetch(`${$admin.accountPortalUrl}/api/auth/reset`, {
+              method: "POST",
+              headers: {
+                "Content-Type": "application/json",
+              },
+              body: JSON.stringify({ email: user.email }),
+            })
+          },
+          extraButtonText: "Resend email",
+        },
+      ])
+    }
+  }
+
+  $: checkVerification(user)
```

Comment:
Since this is just running in a reactive statement, if this retriggered (say because a user updated their name before verifying) would this cause a second banner to appear, or will their only be one? As long as multiple invocations don't flash or cause duplicate banners then we're fine. Otherwise, we'd be better to use run this on mount as soon as the user has loaded.

Reply:
Yeah that's a good point, I'll just run it on mount.

Reply:
Actually, looking at this again, I added a flag that means this is only ran once, and I avoided using the onMount originally because the user object isn't loaded properly when the on mount is ran.

So I think this is fine the way it is 👍